data/notices.json

```diff
@@ -555,6 +555,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(lambda-nodejs): breaking change in esbuild causes import module error",
+      "issueNumber": 30717,
+      "overview": "Use lamda-nodejs module will cause Runtime.ImportModuleError: Cannot find module 'module'. It is from a breaking change ESBuild introduced which changed the default bundling method. This is affecting CDK V2 versions before v2.147.2 and V1 versions later than 1.175.0",
```

Comment:
You should mention the actual construct impacted, as well as the module. Something like this:

```
ESBuild released a breaking change on version <version number> that changed the default bundling behavior. `lambda-nodejs`'s `NodeJsFunction` now to fails to synthesize with the error message `Runtime.ImportModuleError: Cannot find module 'module'`. This issue exists on all CDK versions up to and including `2.147.2`. There is no known workaround at this time. 
```

Is there a workaround? Please correct that if there is.

Reply:
Thanks for the feedback. Updated the overview with workaround.